subsys/dfu/img_util/flash_img.c

```diff
@@ -18,6 +18,7 @@
 #endif
 
 #include <devicetree.h>
+
```

Comment:
remove this empty line from reapplied `[nrf noup] tree-wide: support NCS Partition Manager (PM) definitions.`